packages/mdc-layout-grid/_mixins.scss

```diff
@@ -153,7 +153,7 @@
     align-self: flex-start;
 
     @supports (display: grid) {
-      align-self: start;
+      align-self: flex-start;
```

Comment:
I don't think `flex-start` and `flex-end` make sense when `align-self` is used in the context of CSS Grid (not flexbox). This file should probably be reverted.

The warning is ignorable in this case because there are separate fallback styles for browsers that don't support CSS Grid, which do use `flex-*`.